### DISCUSSION TITLE
Master’s Thesis using BoaviztAPI

### DISCUSSION THREAD

**Discussion by JacobValdemar** [Boavizta/boaviztapi#260] [Show and tell]
Hello community,

I am happy to say that I can finally share my master's thesis with you.
In this thesis I have used cloud instance carbon emission estimates from BoaviztAPI in a cluster autoscaler to minimize the carbon impact of cloud-based Kubernetes clusters. If you are interested in learning how BoaviztAPI was used in this thesis then feel free to check it out. It even contains a chapter just for BoaviztAPI.

https://github.com/JacobValdemar/carbon-efficient-karpenter-thesis

Yours,
Jacob Valdemar

**Comment by samuelrince** [comment]
@criscola you might be interested! 😉

**Reply by criscola** [reply]
I definitely am! Thank you very much!

**Comment by da-ekchajzer** [comment]
Congratulations, it looks like a high-quality piece of work. I'm sharing it with the group and I'm sure there'll be a lot of interest.

**Reply by da-ekchajzer** [reply]
@github-benjamin-davy @tibosmn

**Reply by JacobValdemar** [reply]
Thank you David, I have also put a lot of effort into it. I welcome any questions that you and the rest of the group might have.
